internal/fixtures/online_query_inputs_all_types.json

```diff
@@ -1,70 +1,29 @@
 {
   "all_types.has_many": [
     {
-      "has_many.bool": true,
```

Comment:
Checked this fixture change by hand. This is simply a re-ordering that we actually favor over the old ordering, since it maintains the struct's field order.